__tests__/app.test.js

```diff
@@ -353,4 +353,132 @@ describe("All requests", () => {
+                        expect(article).toHaveProperty("comments_count", expect.any(String));
+                    });
+                });
+        });
+        test("200 will returned and list of articles when all queries are provided except topic query", () => {
+            return request(app)
+                .get("/api/articles?sort_by=title&order=asc")
+                .expect(200)
+                .then(({ body }) => {
+                    const articles = body.articles;
+                    expect(articles).toBeSortedBy("title", {
+                        descending: false,
+                    });
+                    expect(articles).toHaveLength(13);
+                    expect(articles.body).toBeUndefined();
+
+                    articles.forEach((article) => {
+                        expect(article).toHaveProperty("author", expect.any(String));
+                        expect(article).toHaveProperty("title", expect.any(String));
+                        expect(article).toHaveProperty("article_id", expect.any(Number));
+                        expect(article).toHaveProperty("topic", expect.any(String));
+                        expect(article).toHaveProperty("created_at", expect.any(String));
+                        expect(article).toHaveProperty("votes", expect.any(Number));
+                        expect(article).toHaveProperty("article_img_url", expect.any(String));
+                        expect(article).toHaveProperty("comments_count", expect.any(String));
+                    });
+                });
+        });
+        test("200 will returned and list of articles when all queries are NOT provided except sort_by query.", () => {
+            return request(app)
+                .get("/api/articles?sort_by=author")
+                .expect(200)
+                .then(({ body }) => {
+                    const articles = body.articles;
+                    expect(articles).toBeSortedBy("author", {
+                        descending: true,
+                    });
+                    expect(articles).toHaveLength(13);
+                    expect(articles.body).toBeUndefined();
+
+                    articles.forEach((article) => {
+                        expect(article).toHaveProperty("author", expect.any(String));
+                        expect(article).toHaveProperty("title", expect.any(String));
+                        expect(article).toHaveProperty("article_id", expect.any(Number));
+                        expect(article).toHaveProperty("topic", expect.any(String));
+                        expect(article).toHaveProperty("created_at", expect.any(String));
+                        expect(article).toHaveProperty("votes", expect.any(Number));
+                        expect(article).toHaveProperty("article_img_url", expect.any(String));
+                        expect(article).toHaveProperty("comments_count", expect.any(String));
+                    });
+                });
```

Comment:
It would make way more sense here to build up the happy path testing by testing one query at a time, then maybe one test block with a combination of all of the queries.